bucket/ryzen-controller.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "2.5.4",
+    "description": "Utility for tweaking AMD Ryzen laptops.",
+    "homepage": "https://ryzencontroller.com/",
+    "license": {
+        "identifier": "CC0-1.0",
+        "url": "https://gitlab.com/ryzen-controller-team/ryzen-controller/-/blob/master/LICENSE.md"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://gitlab.com/ryzen-controller-team/ryzen-controller/-/jobs/1710820449/artifacts/raw/dist/win/Ryzen%20Controller%20Setup%202.5.4.exe#/dl.7z",
```

Comment:
How do you plan to extract 1710820449?

Reply:
That is why we need to implement `UpdateTracker`.

This package (and many other packages) has to be updated manually. However, in this case, it can be solved by a simple mechanism such as the case of `FossHub`'s hash.

Reply:
I personally am not very supportive of manual updates.

Reply:
> I personally am not very supportive of manual updates.

I agree. Manual updates cost a lot of unnecessary labor. I think we should implement more mechanism to eliminate manual updates as much as possible.

However I don't think it is a reason **not to** add a package. I don't think our solutions has to be "perfect" right from the beginning. We can gradually improve that. ^_^

Reply:
We can merge it later when we support it on stable as well

Reply:
> We can merge it later when we support it on stable as well

Then let's hold this PR until it is finished.